querydsl-mongodb/src/main/java/com/querydsl/mongodb/MongodbSerializer.java

```diff
@@ -127,6 +127,11 @@ public Object visit(Operation<?> expr, Void context) {
             if (subOp == Ops.IN) {
                 return visit(OperationImpl.create(Boolean.class, Ops.NOT_IN, subOperation.getArg(0),
                         subOperation.getArg(1)), context);
+            } else if (subOp == Ops.COL_IS_EMPTY) {
```

Comment:
I meant to detect $or expressions by looking at the nested result and
replacing $or with $nor.
On 19 Mar 2015 00:51, "John Tims" <notifications@github.com> wrote:

> In
> querydsl-mongodb/src/main/java/com/querydsl/mongodb/MongodbSerializer.java
> <https://github.com/querydsl/querydsl/pull/1268#discussion_r26715822>:
>
> > @@ -127,6 +127,11 @@ public Object visit(Operation<?> expr, Void context) {
> >              if (subOp == Ops.IN) {
> >                  return visit(OperationImpl.create(Boolean.class, Ops.NOT_IN, subOperation.getArg(0),
> >                          subOperation.getArg(1)), context);
> > +            } else if (subOp == Ops.COL_IS_EMPTY) {
>
> Can you give me a little more detail? I don't think I understand.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/querydsl/querydsl/pull/1268/files#r26715822>.
>